layouts/partials/openapi/render-object-table.html

```diff
@@ -18,6 +18,7 @@
 {{ $title := .title }}
 {{ $properties := .properties}}
 {{ $required := .required}}
+{{ $additionalProperties := .additionalProperties }}
```

Comment:
please could you add something about this to the comment at the top of the file

Reply:
Sure, see 6487e84c